app.js

```diff
@@ -12,7 +12,56 @@ app.set('views', __dirname + '/views');
 app.use(express.static(__dirname + '/public'));
 
 // setting the spotify-api goes here:
-
+const spotifyApi = new SpotifyWebApi({
+    clientId: process.env.CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET
+  });
+  
+  // Retrieve an access token
+  spotifyApi
+    .clientCredentialsGrant()
+    .then(data => spotifyApi.setAccessToken(data.body['access_token']))
+    .catch(error => console.log('Something went wrong when retrieving an access token', error));
 // Our routes go here:
 
-app.listen(3000, () => console.log('My Spotify project running on port 3000 🎧 🥁 🎸 🔊'));
+
+
+app.get('/', (req, res, next) => {
+    res.render('home')
+});
+
+app.get('/home', (req, res, next) => {
+    res.render('home')
+});
```

Comment:
You should always delete code you are not using. For the 'homepage' , slash is enough, so delete the 2nd route :)